park-dashboard/src/components/NationalParksMap/MapChart.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { memo } from "react";
+import { geoCentroid } from "d3-geo";
+import {
+  ComposableMap,
+  Geographies,
+  Geography,
+  Marker,
+  Annotation,
+} from "react-simple-maps";
+import allStates from "./allStates.json";
+import { Link } from "react-router-dom";
+
+const geoUrl = "https://cdn.jsdelivr.net/npm/us-atlas@3/states-10m.json";
+
+const offsets = {
+  VT: [50, -8],
+  NH: [34, 2],
+  MA: [30, -1],
+  RI: [28, 2],
+  CT: [35, 10],
+  NJ: [34, 1],
+  DE: [33, 0],
+  MD: [47, 10],
+  DC: [49, 21],
+};
+const MapChart = ({
+  stateWiseNationalParks,
+  setTooltipName,
+  setTooltipAddress,
+  setTooltipCity,
+  setTooltipPostCode,
+  setTooltipDescription,
+  setTooltipContact,
+}) => {
+  return (
+    <ComposableMap
+      data-tip=""
+      className="map-container"
+      projection="geoAlbersUsa"
+      style={{ width: "100%", height: "600px" }}
+    >
+      <Geographies geography={geoUrl}>
+        {({ geographies }) => (
+          <>
+            {geographies.map((geo) => (
+              <Geography
+                key={geo.rsmKey}
+                stroke="#FFF"
+                geography={geo}
+                fill="#DAF7A6"
+                onFocus={(e) => {
+                  e.target.style.outline = "none";
+                }}
+              />
+            ))}
+
+            {stateWiseNationalParks &&
+              stateWiseNationalParks.map((park) =>
+                park.nationalParks.map(
+                  (nationalPark, index) =>
+                    nationalPark.coordinates.length === 2 && (
+                      <Link
+                        to={{
```

Comment:
you need to add a unique "key" prop to the child components being rendered in the stateWiseNationalParks mapping. 
here you can add "key={`${park.id}-${index}`}" so it will resolve the key warning.

Reply:
Adding to what @rutujapadgilwar has mentioned, I guess just adding the "key={${park.id}}" would be sufficient to solve this warning, as the park ID is unique anyway.

Reply:
Done, added key.